ieeeuoft/hackathon-template
# Added CSV export functionality using django-import-export library

## Overview

- Resolves #264
- Installs `django-import-export` library
- Adds ability to export the user's `Application` with related fields
- **Note:** *Doesn't* migrate [PR#30](https://github.com/ieeeuoft/makeuoft-site-2021/pull/30/) from the makeuoft-site-2020 - uses `django-import-export` library instead
- **Note:** the `resume_sharing` column was not included since that column is not in the `hackathon-template` models and is unique to the `makeuoft-site-2020` schema.


## Unit Tests Created
- N/A


## Steps to QA
- Make sure to do `pip install -r requirements.txt`
- Create some applications in the admin site(at least 2-3). Go to the admin site `Registration -> Applications` and click on `Export` on the top right. Select the file type you want and then `Submit`. Check to make sure the required columns are included in the file.
